### PR TITLE
[desktop] Translation updated: Italian

### PR DESCRIPTION
**Pull request by alejandroarturom**

### CODE REVIEW

File: desktop/lib/messagebundles/it.po [hunk 1]
```diff
@@ -217,12 +217,12 @@ msgstr "Invia a un amico"
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/bittorrent/BTDownloadActions.java:385
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/library/AbstractLibraryTableMediator.java:219
 msgid "Create New Playlist"
-msgstr "Crea nuova Playlist"
+msgstr "Crea nuova lista di riproduzione"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: this won't fit, anyway to make it shorter?

Comment (gubatron) [reply desktop/lib/messagebundles/it.po]: try `gradle gettextBundle` and then run frostwire so you see how it looks on the Library window, on the left hand side where you can create playlists.

Comment (alejandroarturom) [reply desktop/lib/messagebundles/it.po]: So the option we have is to continue using "playlist"

File: desktop/lib/messagebundles/it.po [hunk 2]
```diff
@@ -2234,14 +2228,12 @@ msgid "New FrostWire Update Available"
 msgstr "Nuova versione disponibile!"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/updates/UpdateManager.java:239
-#, fuzzy
 msgid "FrostWire Team Announcement"
-msgstr "FrostWire per Android"
+msgstr "Annuncio di FrostWire Team"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: `Annuncio della Squaddra FrostWire`?

File: desktop/lib/messagebundles/it.po [hunk 3]
```diff
@@ -2234,14 +2228,12 @@ msgid "New FrostWire Update Available"
...
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/frostwire/gui/updates/UpdateManager.java:252
-#, fuzzy
 msgid "Download Torrent"
-msgstr "Sto scaricando Torrent"
+msgstr "Scarica torrent"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: `Torrent` (Capital T)

File: desktop/lib/messagebundles/it.po [hunk 4]
```diff
@@ -2476,34 +2464,29 @@ msgid "Support FrostWire development with a Bitcoin donation"
...
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/DonationButtons.java:50
 #, fuzzy
 msgid "Support FrostWire development with a ZCash donation"
 msgstr "Sostenere lo sviluppo FrostWire con una donazione Paypal"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/DonationButtons.java:51
-#, fuzzy
 msgid "Support FrostWire development with a Bitcoin Gold donation"
-msgstr "Sostenere lo sviluppo FrostWire con una donazione Bitcoin"
+msgstr "Sostenere lo sviluppo FrostWire con una donazione Zcash"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: `Bitcoin Gold` (not ZCash if I read correctly above)

File: desktop/lib/messagebundles/it.po [hunk 5]
```diff
@@ -2476,34 +2464,29 @@ msgid "Support FrostWire development with a Bitcoin donation"
...
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/DonationButtons.java:49
-#, fuzzy
 msgid "Support FrostWire development with a Litecoin donation"
-msgstr "Sostenere lo sviluppo FrostWire con una donazione Bitcoin"
+msgstr "Sostenere lo sviluppo FrostWire con una donazione Litecoin"
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/DonationButtons.java:50
 #, fuzzy
 msgid "Support FrostWire development with a ZCash donation"
 msgstr "Sostenere lo sviluppo FrostWire con una donazione Paypal"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: Here's where `ZCash` should go

File: desktop/lib/messagebundles/it.po [hunk 6]
```diff
@@ -2896,28 +2871,24 @@ msgstr ""
...
+msgstr "<b>Seminando</b><p>download torrent completati."
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:223
-#, fuzzy
 msgid ""
 "<b>Not Seeding</b><p>File chunks might be shared only during a torrent "
 "download.</p>"
 msgstr ""
-"<b>Non in Seeding</b>.<p>Porzioni di file possono essere condivise solo "
+"<b>Non seminando</b>.<p>Porzioni di file possono essere condivise solo "
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: `Seminando` (capital `S`)

File: desktop/lib/messagebundles/it.po [hunk 7]
```diff
@@ -2896,28 +2871,24 @@ msgstr ""
...
 "durante<p>un download di torrent."
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/StatusLine.java:468
-#, fuzzy
 msgid "Not Seeding"
-msgstr "Distribuzione seme"
+msgstr "Non seminando"
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: `Non Seminando` (capital S)

Comment (alejandroarturom) [reply desktop/lib/messagebundles/it.po]: Capital letters work differently than in English. "No se está compartiendo", no debería ser "No se está Compartiendo"

File: desktop/lib/messagebundles/it.po [hunk 8]
```diff
@@ -5173,7 +5136,7 @@ msgstr ""
 
 #: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/search/SearchOptionsPanel.java:136
 msgid "Name|Source|Ext."
-msgstr ""
+msgstr "Nome | Fonte | Ext."
```
Comment (gubatron) [inline desktop/lib/messagebundles/it.po]: remove extra spaces -> `Nome|Fonte|Ext.`

File: desktop/lib/messagebundles/it.po (hunk 1 above)
Comment (alejandroarturom) [reply desktop/lib/messagebundles/it.po]: Do you prefer playlist instead of lista di riproduzione?

Comment (gubatron) [reply desktop/lib/messagebundles/it.po]: In this instances I think it makes more sense, but then, if you do it here, you should probably do it everywhere else for consistency.

File: desktop/lib/messagebundles/it.po (hunk 8 above)
Comment (gubatron) [reply desktop/lib/messagebundles/it.po]: this one is still missing

**Pull request merged**